gulpfile.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
First, you can always use `mocha` directly. Second npm scripts allow extra commands to punch to the command via `--`. So, for example, 

```sh
$ npm run test-only -- --grep=foo --ANY_OTHER_MOCHA_CLI_FLAG
```

works just fine.

Also, we write our npm package scripts to be the portable subset of shell commands that is universally (mac, win, linux) compatible.